Signal/src/call/CallService.swift

```diff
@@ -376,6 +378,8 @@ protocol CallServiceObserver: class {
         }
 
         fulfillReadyToSendIceUpdatesPromise()
+        self.fulfillReadyToSendIceUpdatesPromise = nil
```

Comment:
Why do we bother to nil all these promise methods out? I don't *think* there's harm in removing them, but we already have a central place to `reset` state - in `terminate`.

What's the danger in these lingering?